google-cloud-apis/src/com/google/cloud/tools/intellij/cloudapis/CloudApiManagementConfirmationDialog.java

```diff
@@ -123,6 +126,9 @@
             GoogleCloudApisMessageBundle.message(
                 "cloud.apis.management.dialog.serviceaccount.header")));
 
+    infoLabel.setIcon(General.Information);
```

Comment:
just to confirm - looks like you are setting the icon in code now (instead of I assume in the form like before). Is that something needed for the fix?

Reply:
yeah, it's impossible to set icon directly as the form editor supports PNG/GIF or other pixel formats, and they now use SVG which must be loaded first.

Reply:
got it. thanks